lib/libutee/include/tee_api.h

```diff
@@ -33,6 +33,9 @@
 #include <compiler.h>
 #include <tee_api_defines.h>
 #include <tee_api_types.h>
+#if defined(CFG_TEE_PANIC_DEBUG)
```

Comment:
+1 for enabling this by default when `CFG_TEE_TA_LOG_LEVEL >= 3` (debug), we could do this in `mk/config.mk`, but I would still keep the new flag, because it's more flexible (one config flag per feature especially when there is no "hard" dependencies).